TweenController/TweenPromise.swift

```diff
@@ -36,43 +36,43 @@ public struct TweenPromise<T:Tweenable> {
     let resolvedDescriptors: [TweenDescriptor<T>]
     weak var registration: DescriptorRegistration?
     
-    public func to(_ to: T, at progress: Double, withEasing easing: Easing.Function = Easing.linear) -> TweenPromise<T> {
+    public func to(_ to: T, at progress: Double, withEasing easing: @escaping Easing.Function = Easing.linear) -> TweenPromise<T> {
         assert(progress != self.progress, "'to' progress must be different than 'from' progress")
         
         let descriptor = TweenDescriptor(fromValue: from, toValue: to, interval: self.progress..<progress, easingFunction: easing)
-        registration?.register(descriptor: descriptor)
+        registration?.register(descriptor)
         return TweenPromise(from: to, progress: progress, resolvedDescriptors: resolvedDescriptors + [descriptor], registration: registration)
     }
     
-    public func then(to: T, at progress: Double, withEasing easing: Easing.Function = Easing.linear) -> TweenPromise<T> {
+    public func then(_ to: T, at progress: Double, withEasing easing: @escaping Easing.Function = Easing.linear) -> TweenPromise<T> {
```

Comment:
I think the argument label should still exist here. This reads more naturally:
    
    controller.tween(from: a, at: 0.0)
        .to(b, at: 0.5)
        .then(to: c, at: 1.0)
        .with(action: doTheThing)

as apposed to:

    tweenController.tween(a, at: 0.0)
            .to(b, at: 0.5)
            .then(c, at: 1.0)
            .with(doTheThing)